pkgs/os-specific/linux/asus-wmi-sensors/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, kernel }:
+
+stdenv.mkDerivation rec {
+  name = "asus-wmi-sensors-${version}-${kernel.version}";
+  version = "0.0.1";
```

since you're collecting from an unofficial revision rather than release
```suggestion
  version = "unstable-2019-11-07";
```